docs/Create-Packages/Creating-a-Package.md

```diff
@@ -236,22 +236,20 @@ When using a convention-based working directory, on the other hand, NuGet assume
 
 Such a folder structure also allows you to easily include files that might not be part of a project at all, including:
 
-- Content and source code that should be injected into the target project.
-- PowerShell scripts (packages used in NuGet 2.x can include installation scripts as well, which is not supported in NuGet 3.x and later).
-- Transformations to existing configuration and source code files in a project.
+- Content and source code that should be available for reference across projects through global packages folder.
+- PowerShell scripts only initialization script init.ps1 (packages used with NuGet packages.config format can include installation scripts as well, which is not supported in NuGet project.json and packageReference formats).
+- Transformations to source code files in a project via .pp files.
 
 The folder conventions are as follows:
 
 | Folder | Description | Action upon package install |
 | --- | --- | --- |
 | (root) | Location for readme.txt | Visual Studio displays a readme.txt file in the package root when the package is installed. |
-| lib | Assembly (`.dll`), documentation (`.xml`), and symbol (`.pdb`) files | Assemblies are added as references; `.xml` and `.pdb` copied into project folders. See [Supporting multiple target frameworks](Supporting-Multiple-Target-Frameworks.md) for creating framework target-specific sub-folders. |
-| runtimes | Architecture-specific assembly (`.dll`), symbol (`.pdb`), and native resource (`.pri`) files | Assemblies are added as references; other files are copied into project folders. See [Supporting multiple target frameworks](Supporting-Multiple-Target-Frameworks.md). |
-| content | Arbitrary files | Contents are copied to the project root |
-| build | MSBuild `.targets` and `.props` files | Automatically inserted into the project file (NuGet 2.x) or `project.lock.json` (NuGet 3.x+). |
-| tools | Powershell scripts and programs accessible from the Package Manager Console | Contents are copied to the project folder, and the `tools` folder is added to the PATH environment variable. |
-
-Think of the **content** folder as the root of the target application. To have the package add an image in the application's */images* folder, place it in the package's *content/images* folder.
+| lib | Assembly (`.dll`), documentation (`.xml`), and symbol (`.pdb`) files | Assemblies are added as references; `.xml` and `.pdb` are extracted in packages folder. See [Supporting multiple target frameworks](Supporting-Multiple-Target-Frameworks.md) for creating framework target-specific sub-folders. |
```

Comment:
I'm still confused by `extracted in packages folder`.  Maybe it should just say that additional package files are extracted to disk such as .xml and .pdb files.

Reply:
how about "other files in this folder are extracted to disk in global packages folder?